src/style.css

```diff
@@ -1,3 +1,80 @@
-body {
+* {
```

Comment:
Using the wildcard is allowed only for box-sizing reset, so try to reset margins and padding wisely and where it is needed.